### PR TITLE
Remove BlocklyProp Client code support

### PR DESCRIPTION
**Pull request by zfi**
Addresses issue #571

### CODE REVIEW

File: src/modules/client_connection.js [hunk 1]
```diff
@@ -420,12 +402,12 @@ function parseCompileMessage(message) {
  * Lost websocket connection, clean up and restart findClient processing
  */
 function lostWSConnection() {
-  if (clientService.type !== serviceConnectionTypes.HTTP) {
-    clientService.loaderResetDetect = true;
-    clientService.activeConnection = null;
-    clientService.type = serviceConnectionTypes.NONE;
-    clientService.available = false;
-  }
+  // if (clientService.type !== serviceConnectionTypes.HTTP) {
```
Comment (pjewald) [inline src/modules/client_connection.js:405]: Remove these comments.

File: src/modules/client_service.js [hunk 2]
```diff
@@ -36,13 +36,15 @@ const debug = false;
...
  *    NONE: string,
- *    HTTP: string,
  *    WS: string
  *  }}
  */
 export const serviceConnectionTypes = {
   // Constants for the type property
-  HTTP: 'http',
+
+  // BP Client is deprecated
```
Comment (pjewald) [inline src/modules/client_service.js:45]: Remove these comments

File: src/modules/client_service.js [hunk 3]
```diff
@@ -423,26 +426,27 @@ export const clientService = {
...
      */
     RECOMMENDED: '1.0.1',
 
-    /**
-     * {string} Constant Semantic versioning, Minimum client/launcher version
-     * supporting coded/verbose responses.
-     * NOTE: (remove after MINIMUM_ALLOWED > this)
-     */
-    CODED_MINIMUM: '0.7.5',
+    // /**
```
Comment (pjewald) [inline src/modules/client_service.js:437]: Remove these comments. They are no longer required.

File: src/modules/client_service.js [hunk 4]
```diff
@@ -547,10 +547,11 @@ export function initTerminal() {
       document.getElementById('serial_console'),
 
       function(characterToSend) {
-        if (clientService.type === serviceConnectionTypes.HTTP &&
-            clientService.activeConnection) {
-          clientService.activeConnection.send(btoa(characterToSend));
-        } else if (clientService.type === serviceConnectionTypes.WS) {
+        // if (clientService.type === serviceConnectionTypes.HTTP &&
```
Comment (pjewald) [inline src/modules/client_service.js:550]: Remove comments.

**Review by pjewald — CHANGES_REQUESTED** [review]
Please see inline notes to complete the PR.

File: src/modules/client_connection.js (hunk 1 above)
Comment (zfi) [reply src/modules/client_connection.js:405]: Completed in the next push.

File: src/modules/client_service.js (hunk 2 above)
Comment (zfi) [reply src/modules/client_service.js:45]: Completed in the next push..

File: src/modules/client_service.js (hunk 3 above)
Comment (zfi) [reply src/modules/client_service.js:437]: Completed in the next push.

File: src/modules/client_service.js (hunk 4 above)
Comment (zfi) [reply src/modules/client_service.js:550]: Completed in the next push.

**Review by pjewald — APPROVED** [review]
/LG2M

**Comment by zfi** [comment]
This completes work on #571.

**Pull request merged**